package.json

```diff
@@ -58,6 +58,7 @@
     "@testing-library/react": "^14.0.0",
     "@testing-library/user-event": "^14.4.3",
     "@types/jest": "^29.5.3",
+    "@types/qlik-engineapi": "^12.67.13",
```

Comment:
Why did you need to add this? Was it not update to date in `@nebula.js/stardust`?

Reply:
seems like that was required for type support on: 
1. `EngineAPI.IGenericHyperCubeLayout`
2. `EngineAPI.IApp`
3. `EngineAPI.IGenericObject`
4. `EngineAPI.IField`

which i need them for typing

Reply:
Looks like an issue with `pnpm` to me. It does not install `@types/qlik-engineapi` for some reason even tho it's a dependency in `@nebula.js/stardust` (which is installed..)

Reply:
I think we can ignore it for now. Also we should "pin" all versions in another PR